web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -264,6 +264,7 @@ export default function Dashboard() {
   ];
   let defaultPage = URLS.STATS;
   if (isChallengeEnrolled) {
+    // @ts-ignore
```

Comment:
I truly can't be bothered. Seeing the following errors:

```
web        | [FE] ERROR in [at-loader] ./src/components/pages/dashboard/dashboard.tsx:267:46
web        | [FE]     TS2322: Type 'ComponentClass<Pick<Props & RouteComponentProps<any, StaticContext, any>, "dashboardLocale" | "allGoals">, any> & WithRouterStatics<(props: Props & RouteComponentProps<any, StaticContext, any>) => Element>' is not assignable to type '({ allGoals, dashboardLocale }: Props) => Element'.
web        | [FE]   Type 'ComponentClass<Pick<Props & RouteComponentProps<any, StaticContext, any>, "dashboardLocale" | "allGoals">, any> & WithRouterStatics<(props: Props & RouteComponentProps<any, StaticContext, any>) => Element>' provides no match for the signature '({ allGoals, dashboardLocale }: Props): Element'.
```

It works as expected, but doesn't like that we've tacked the new React Router props onto `ChallengePage`. This is a problem for another day.

Reply:
I'm good with it if you are.